search-service/src/test/kotlin/com/egm/stellio/search/service/EntityEventListenerServiceTest.kt

```diff
@@ -280,6 +386,18 @@ class EntityEventListenerServiceTest {
         }
     }
 
+    private fun prepareAttributeAppendEventPayload(payload: String): String =
```

Comment:
if you make `operationType` a function parameter, you should be able to have to just one `prepareXXXEventPayload` function instead of 3?

Reply:
Indeed